attributes/default.rb

```diff
@@ -342,31 +342,24 @@
 # * `false` to explicitly disable it
 # Leave it to `nil` to let the agent's default behavior decide whether to run the process-agent
 default['datadog']['enable_process_agent'] = nil
+default['datadog']['process_agent']['url'] = 'https://process.datadoghq.com'
 
-# Comma-separated list of regex patterns matching process commands to blacklist.
-# Example: 'my-secret-app,dbpass'
-default['datadog']['process_agent']['blacklist'] = nil
-
-# Comma-separated list of regex patterns of containers to include or skip.
-# Each pattern should be in the form of "field:pattern" where 'field' is either
-# 'image' or 'name'.
-# Example: 'image:redis,image:nginx'
-default['datadog']['process_agent']['container_blacklist'] = nil
-# Whitelist is applied after the blacklist.
-default['datadog']['process_agent']['container_whitelist'] = nil
```

Comment:
Blacklists for containers are handled in the standard Agent 6 way as we share the tagger code.

Reply:
We should also at some point clean up the config blacklist code in the process-agent since it looks like it's never used: https://github.com/DataDog/datadog-process-agent/blob/master/config/config.go#L218